NSS-Day-Cohort-68/rare-client-monty-py
# Feature/view tags

Ticket https://github.com/NSS-Day-Cohort-68/rare-api-monty-py-api/pull/36 View Tags

Purpose: To allow a user to view all the tags in the database.

What I Changed: I created a route, new service function, a component file and the navbar item for Tag Management. 

Testing

1. Run the API debugger and the npm start on the client. 
2. Login to with a user email and click the tag management navbar item.
3. You should see a list of all tags with a button that links to the create tag page.
